tests/query_test.py

```diff
@@ -305,6 +305,14 @@ def test_time(self):
     time = "1970-01-01T00:00:00.123456789Z"
     assert self._q(query.time(time)) == FaunaTime(time)
 
+    # 'now' refers to the current time.
+    now = self._q(query.time('now'))
+    assert 0 < (datetime.now() - now.to_datetime()).microseconds < 1000000
```

Comment:
Matt just said what I was writing a comment about, but I will point out stuff that's wrong with the code assuming time is perfect between machine (which it's not), for future reference

`microseconds` is only the microseconds portion of the delta. You would have to use `total_seconds()` instead to get the actual full difference. Also, `total_seconds()` will be negative if the local clock is running ahead. Also, I believe you would have to use `.utcnow()` as the dates returned from fauna are UTC (unless the comparison is aware of the timezone difference?)